products/silverstripe.md

```diff
@@ -74,6 +74,41 @@ releases:
     latest: "4.10.11"
     latestReleaseDate: 2022-11-21
 
+-   releaseCycle: "3.7"
+    releaseDate: 2018-06-08
+    support: true
+    eol: 2021-09-30
+    latest: "3.7.7"
+    latestReleaseDate: 2021-06-02
+
+-   releaseCycle: "3.1"
```

Comment:
No release for 3.2 to 3.6 ?

Reply:
There are, but they follow semantic versioning and are irrelevant in that sense and would only clutter the page.

Silverstripe adopted semantic versioning in 3.2. That's why 3.0 and 3.1 are still relevant (because they need a manual update). From 3.2 onwards the minor versions become less significant.

Reply:
Thanks for the explanation.